### DISCUSSION TITLE
osu!lazer stopped starting after installing update 2021.703.0

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#13750] [Q&A] [answered]
So, today I decided to start osu! Lazer as usual, but after I left the game to restart it after installing update 2021.703.0, I saw a message from Windows "osu! has stopped working". I am using Windows 7 64-bit, so I guess the issue is not with compatibility. What should I do to fix the problems? 
[database.log](https://github.com/ppy/osu/files/6756118/database.log)
[network.log](https://github.com/ppy/osu/files/6756120/network.log)
[performance.log](https://github.com/ppy/osu/files/6756121/performance.log)
[runtime.log](https://github.com/ppy/osu/files/6756122/runtime.log)
[updater.log](https://github.com/ppy/osu/files/6756123/updater.log)

**Comment by bdach** [comment, marked as answer]
#13769
